REVIEWING.md

```diff
@@ -0,0 +1,45 @@
+# SSS Reviewer Guidelines
+
+These are recommendations for reviewing contributions to SSS repositories.
+Reviewing is important to ensure high quality and consistency of content.
+As a reviewer, we trust you will vet contributions and provide constructive suggestions to update them to fit the SSS guidelines.
+
+## Activities / Challenges
+
+Make sure that activities / challenges are of good quality:
+
+- Difficulty is properly set.
+- The topic of the activity / challenge is appropriate.
+- The provided solution is working.
+- Vulnerability and explanations are easy to understand.
+
+## Assets
+
+Check if all of them are used and mentioned in the content.
+If the assets are not original, make sure their license is compatible to [ours](./COPYING.md) and that their source is properly cited.
+Verify that their names are appropriate and respect the following rules:
+
+- Use dash between words (e.g. `image-name.png`).
+- Use only lowercase ASCII characters.
+
+## Content
+
+Make sure that the content is of good quality:
+
+- It follows the general structure of a [session](https://github.com/security-summer-school/template-public/tree/master/session-name-1).
+- Assets are displayed well.
+- The content has at most one sentence per line.
+- The content is easy to understand.
+- Wording and general spelling are correct.
+- Proper formatting is used (such as bold, italic, typewriter formats).
+
+## Working on Pull Requests
+
+You will review contributions as pull requests, following the [PR review guidelines from GitHub](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/about-pull-request-reviews).
+Contributions must follow the [contributing guidelines](CONTRIBUTING.md).
```

Comment:
```suggestion
Contributions must follow the [contributing guidelines](./CONTRIBUTING.md).
```

Reply:
By default it will use this path, no need to specific exact path.

Reply:
Then change the one on line 19 for consistency

Reply:
Done.